app/views/admin/_admin_nav.html.erb

```diff
@@ -7,6 +7,9 @@
     <%= span_if_current ts('Post AO3 News', key: 'header'), new_admin_post_path %>
   </li>
   <li>
+    <%= link_to t('admin.admin_nav.delete', default: 'Delete Post'), @admin_post, confirm: 'Are you sure you want to delete this news post?', method: :delete %>
```

Comment:
I'm not 100% sure, but I _think_ this navigation is included on pages other than the edit form, e.g. the index that lists all the admin posts and the known issues page, so including the Delete button here is going to cause an error on those pages.